ci/infra/azure/worker-instance.tf

```diff
@@ -3,28 +3,32 @@ resource "azurerm_network_interface" "worker" {
   name                = "${var.stack_name}-worker-${count.index}-nic"
   location            = azurerm_resource_group.resource_group.location
   resource_group_name = azurerm_resource_group.resource_group.name
+  depends_on          = [azurerm_subnet.nodes]
```

Comment:
same here

Reply:
We only have template on v5 beta, therefore this PR will update it.